### PR TITLE
Hide Confirmation Alert on Form Submission

### PR DESCRIPTION
**Pull request by zainab-amir**
Fixed:
- If an error occurs on sign in page after password reset request,
the confirmation message is not dismissed.

Screen Shot 2021-02-08 at 12 40 23 PM

[VAN-347](https://openedx.atlassian.net/browse/VAN-347)

### CODE REVIEW

File: src/login/LoginPage.jsx [hunk 1]
```diff
@@ -195,7 +196,7 @@ class LoginPage extends React.Component {
               {this.props.loginError ? <LoginFailureMessage loginError={this.props.loginError} /> : null}
               {this.state.isSubmitted ? window.scrollTo({ left: 0, top: 0, behavior: 'smooth' }) : null}
               {activationMsgType && <AccountActivationMessage messageType={activationMsgType} />}
-              {this.props.forgotPassword.status === 'complete' ? <ConfirmationAlert email={this.props.forgotPassword.email} /> : null}
```
Comment (waheedahmed) [inline src/login/LoginPage.jsx:198]: Can we simply add another check in this condition like this?
```
{this.props.forgotPassword.status === 'complete' && !this.props.loginError ? <ConfirmationAlert email={this.props.forgotPassword.email} /> : null}
```

**Review by waheedahmed — CHANGES_REQUESTED** [review]

File: src/login/LoginPage.jsx (hunk 1 above)
Comment (zainab-amir) [reply src/login/LoginPage.jsx:198]: yes. let me update the PR

File: src/reset-password/tests/ResetPasswordPage.test.jsx [hunk 2]
```diff
@@ -11,7 +11,6 @@ import { resetPassword } from '../data/actions';
 
 import ResetPasswordPage from '../ResetPasswordPage';
 
-jest.mock('../data/selectors', () => jest.fn().mockImplementation(() => ({ resetPasswordSelector: () => ({}) })));
```
[the remark this answers is not in the document]
Comment (zainab-amir) [reply src/reset-password/tests/ResetPasswordPage.test.jsx:14]: I think this was a mistake but I don't think we are using it. Should I bring the statement back?

[the remark this answers is not in the document]
Comment (waheedahmed) [reply src/reset-password/tests/ResetPasswordPage.test.jsx:14]: If it doesn't affect the tests then no need.

**Review by waheedahmed — APPROVED** [review]
👍 once test passes.

**Pull request merged**